### PR TITLE
Update lstm_generator_cityname.py

### PR DESCRIPTION
**Pull request by hoondy**
1. SSL issue with URL call
2. Convert city names to lower case

### CODE REVIEW

File: examples/nlp/lstm_generator_cityname.py [hunk 1]
```diff
@@ -1,19 +1,21 @@
...
 if not os.path.isfile(path):
-    urllib.request.urlretrieve("https://raw.githubusercontent.com/tflearn/tflearn.github.io/master/resources/US_Cities.txt", path)
+    context = ssl._create_unverified_context()
+    moves.urllib.request.urlretrieve("https://raw.githubusercontent.com/tflearn/tflearn.github.io/master/resources/US_Cities.txt", path, context=context)
 
 maxlen = 20
 
 X, Y, char_idx = \
-    textfile_to_semi_redundant_sequences(path, seq_maxlen=maxlen, redun_step=3)
+    textfile_to_semi_redundant_sequences(path, seq_maxlen=maxlen, redun_step=3, to_lower_case=True)
```
Comment (aymericdamien) [inline examples/nlp/lstm_generator_cityname.py]: Please remove to_lower_case-True, as it is interesting to see that the model can learn by itself to use upper-case letters in a city name (Usually the first letter).

Comment (aymericdamien) [inline examples/nlp/lstm_generator_cityname.py]: Please remove to_lower_case-True, as it is interesting to see that the model can learn by itself to use upper-case letters in a city name (Usually the first letter).

**Comment by fresiy8361** [comment]
Thanks for this patch, could you take a look at my comment and update your code? Then I will merge the pull request, thanks :)

**Comment by aymericdamien** [comment]
Thanks for this patch! Could you please check my comment and update your code? I will then merge the PR :)

File: examples/nlp/lstm_generator_cityname.py (hunk 1 above)
Comment (hoondy) [inline examples/nlp/lstm_generator_cityname.py]: I thought it was more important to limit the number of characters, as upper case adds another 26 characters into its dictionary. In my opinion, the first letter upper case is more or less mechanical and can be applied after the model is trained.

**Comment by aymericdamien** [comment]
Looks ok now, thanks :)

**Pull request merged**